docs/content/latest/yugabyte-platform/security/enable-encryption-at-rest.md

```diff
@@ -33,54 +33,56 @@ For details on the features, assumptions, design, data key management, universe
+    - **Equinix SmartKey:** Once the universe has started being created, a security object will be created with SmartKey through REST API with the universe UUID as the name of the security object. This object will then be exported to the Yugabyte Platform as the universe key. Once received, the Yugabyte Platform sends the key to all masters to persist. Yugabyte Platform does not persist any information about the security object beyond a reference to the object `kId`. At this point, data within the universe will be encrypted at rest using data keys encrypted with the aforementioned universe key generated through the selected KMS configuration. For more information, see [Create a KMS configuration using Equinix SmartKey ](https://docs.yugabyte.com/latest/yugabyte-platform/security/create-kms-config.md/equinix-smartkey/).
+    - **AWS KMS:** Once the universe has been created with encryption at rest enabled, Platform persists the ciphertext of the universe key (because AWS does not persist any CMK-generated data keys themselves) and requests the plaintext of the universe key from AWS KMS using the KMS configuration whenever it needs to provide the universe key to the master nodes. able to specify the KMS endpoint so they can ensure that the encryption  traffic is being routed across their internal links and not going across the external network. For more information, see [Create a KMS configuration using AWS KMS](https://docs.yugabyte.com/latest/yugabyte-platform/security/create-kms-config.md/aws-kms/).
 
 ## Enable encryption at rest on an existing universe
 
-To enable encryption at rest on an existing universe:
+You can enable encryption at rest on an existing universe as follows:
 
 1. Open the Yugabyte Platform console and click **Universes**.
 
 2. Select the universe for which to enable encryption.
 
-3. Select **Edit Universe > Edit Security > Encryption-at-Rest**. 
+3. Select **Actions > Edit Universe > Edit Security > Encryption-at-Rest**. 
 
-4. Deselect **Enable Encryption-at-Rest for <UNIVERSE_NAME>** and click **Submit**.
+4. In the **Manage Encryption at-Rest** dialog, toggle **Enable Encryption-at-Rest for <Universe_name>**.
 
    When the encryption is enabled, the **Key Management Service Config** option appears.
 
 5. Select your KSM configuration from the **Key Management Service Config** list. The list displays only preconfigured KMS configurations. If you need to create one, see [Create a KMS configuration](../create-kms-config).
 
-6. Verify that the `EnableEncryptionAtRest` task completed successfully.
+6. Click **Submit**.
+
+7. Verify that the `EnableEncryptionAtRest` task completed successfully.
 
 ## Back up and restore data from an encrypted at rest universe
 
```

Comment:
add "Select Actions > Edit Universe" to steps in Rotate and Disable sections?
also missing "In the Manage Encryption at-Rest dialog, " in step 3 of disable section?